docs/src/install/database.rst

```diff
@@ -101,15 +98,15 @@ Alternatively, here's an example including all three configuration methods.
    $ orion db test --config local.yaml
```

Comment:
Here the text should be adjusted. Perhaps make it an example using mongodb, because there isn't much options to set on pickleddb.